src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/Given_ProcessType.cs

```diff
@@ -122,26 +75,8 @@ public void Process_BasePriorityUnix()
             }
         }
 
-        public void Sleep(double delayInMs)
-        {
-            Task.Delay(TimeSpan.FromMilliseconds(delayInMs)).Wait();
-        }
-
-        public void Sleep()
-        {
-            Sleep(50D);
-        }
-
-        public void StartAndKillProcessWithDelay(Process p)
-        {
-            p.Start();
-            Sleep();
-            p.Kill();
-            Assert.True(p.WaitForExit(WaitInMS));
-        }
-
         [Fact]
-        public void Process_EnableRaiseEvents()
+        public void Then_EnableRaiseEventsShouldBe()
```

Comment:
The GivenWhenThen naming seems strange in cases like this.  Most of these test methods are setting up the When (in some cases even setting up the Given, though most of that's done by the initialization earlier), yet the method is named Then_*ShouldBe.  Strange, at least to me.